### ISSUE TITLE
Feature request: List items with Local metadata

### ISSUE THREAD

**Issue by BEisem** [Taxel/PlexTraktSync#283]
### Feature description

Older versions of the script would list items that have no match on TMDB, and would return the error `Provider local not supported.` I know it was not supposed to report this, and the intention was that it would eventually silently skip these items. Now that these listings have been removed, I'm finding it was actually quite helpful in being able to find these items, so I could either add them to TMDB or Fix Match in Plex so that they link correctly.

Now that this notification has been removed, I realize it would be great if another command line option was added to the script, that would again list these items. Perhaps it could be totally separate from the sync command, but essentially it would just output any items that only have local metadata.

### Use case

As I said above, in older versions, it was listing all the TV shows and/or movies that only have local metadata, which meant they were not matching with anything on TMDB. As a result, it was a handy way of being able to find items that needed to either be fixed, or that might be missing from TMDB.

### Workarounds

Use an older version of the script (i.e. 0.7.x)

**Comment by glensc** [comment]
This is already done as you your request! 
- https://github.com/Taxel/PlexTraktSync/pull/276

From the previous issue you created! 
- https://github.com/Taxel/PlexTraktSync/issues/268

**Comment by BEisem** [comment]
No, this is not the same as I was requesting before... I tried using the debug logs, but because it is a debug log, it is cluttered with so many other things it's next to impossible to find these items I was looking for. In this feature request, I wasn't asking for log output, I was asking for console output if you issue a certain command (the same way it was appearing in the console output for the sync command in 0.7.x). It's ok though, I still have the 0.7.x version which I can use for this purpose whenever I need it.

I'm going to take a step back from testing this project, because frankly I'm tired of being berated and yelled at by you. You are, frankly, one of the rudest, most arrogant people I have ever dealt with. Pretty much every comment or post I have made has been met with derision, snarky replies, or worse. I've kept my mouth shut until now, but I'm tired of it.

Seeing as how I was the only person actively testing this for you, I guess you'll just have to test it yourself from now on. I'm tired of being treated this way, and honestly I don't deserve it.

**Comment by twolaw** [comment]
I agree with BEisem : #276 should be at a warning level instead of debug. This is important information for user.

@glensc wrote :
> "Provider local not supported" should not be displayed

I think it should be displayed as warning. The Plex libraries chosen by user to be synced with Trakt are supposed to be scraped by an external provider.

**Comment by glensc** [comment]
Well, he asked for debug, so I made it debug.

**Comment by glensc** [comment]
Somewhy the issue close on pull request merge doesn't work in this repo. Closing manually after merge.

**Comment by BEisem** [comment]
> Well, he asked for debug, so I made it debug. He never said debug is not satisfactory only after I had closed the issue as his request was fulfilled but he did not notice it.

No, I never asked for debug. You said it was a copy-paste error that caused it to be displayed, and it was going to be removed. I asked you not to remove it, and in the end I settled for debug, because that seemed to be my only option. But after using it for a bit, I realized it wasn't satisfactory so I asked for a better option.

Again, you seem to take every opportunity to take a shot at me, and I don't understand why. It's really pretty ridiculous.
